### ISSUE TITLE
Uninitialized Constant Apartment::Tenant on Rails 4.2

### ISSUE THREAD

**Issue by mdubbs** [influitive/apartment#194]
Rails 4.2
Ruby 2.1.5p273

`Apartment` seems to work, however `Apartment::Tenant` returns the error below. I don't think it's a setup error on my part, but it's always possible! :laughing: 

```
2.1.5 :001 > Apartment
 => Apartment
2.1.5 :002 > Apartment::Tenant
NameError: uninitialized constant Apartment::Tenant
	from (irb):2
	from /Users/mwilliams/.rvm/gems/ruby-2.1.5/gems/railties-4.2.0/lib/rails/commands/console.rb:110:in `start'
	from /Users/mwilliams/.rvm/gems/ruby-2.1.5/gems/railties-4.2.0/lib/rails/commands/console.rb:9:in `start'
	from /Users/mwilliams/.rvm/gems/ruby-2.1.5/gems/railties-4.2.0/lib/rails/commands/commands_tasks.rb:68:in `console'
	from /Users/mwilliams/.rvm/gems/ruby-2.1.5/gems/railties-4.2.0/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/mwilliams/.rvm/gems/ruby-2.1.5/gems/railties-4.2.0/lib/rails/commands.rb:17:in `<top (required)>'
	from /Users/mwilliams/rackfleet/bin/rails:8:in `<top (required)>'
	from /Users/mwilliams/.rvm/rubies/ruby-2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/mwilliams/.rvm/rubies/ruby-2.1.5/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
```

**Comment by achr3f** [comment]
+1 

for me i can't update to rails 4.2 due to lock version of `apartment`
```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    apartment (>= 0) ruby depends on
      activerecord (< 4.2, >= 3.1.2) ruby

    rails (= 4.2.0) ruby depends on
      activerecord (4.2.0)
```

**Comment by JanStevens** [comment]
Can we do a small version release that fixes this issues without the big changes of renaming `switch` to `switch!` and `process` to `switch`?

**Comment by navaneeth** [comment]
I am also facing the same issue. Can't use apartment because of this. When is the next release planned?

**Comment by bradrobertson** [comment]
I'll release a patch version for Rails 4.2. I still want to test a few things out before the new interface changes get pushed

**Comment by bradrobertson** [comment]
Update pushed. This shouldn't have any affect on the unknown constant though, not sure what could be causing that.

**Comment by achr3f** [comment]
thanks,  it work for me :)

**Comment by mdubbs** [comment]
This seems to have been resolved with 0.26.0, so I am going to close it.
